android/app/src/main/java/io/peggyjo/peggymeter/HistoryTextFragment.java

```diff
@@ -31,12 +38,44 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         dataController.addMoodListener(this);
         // Inflate the layout for this fragment
         View view = inflater.inflate(R.layout.fragment_history_text, container, false);
+        table = view.findViewById(R.id.LOG_ENTRIES_CONTAINER);
+        table.setOnItemLongClickListener(new AdapterView.OnItemLongClickListener() {
+            @Override
+            public boolean onItemLongClick(AdapterView<?> parent, View view, int position, long id) {
+                //used instead of getSelectedItem because getSelectedItem causes a null pointer exception
```

Comment:
Please use the following comment formatting:
// Lorem ipsum.
// Dolor sit
// amet.

Reply:
Anyway consider dropping this comment, this looks like the right way

Reply:
It's been removed from my code.